anchore-ci/local_ci

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
I think this file should have a more descriptive name then `local_ci` like `run_ci_command` or `ci_command_wrapper`. I want to be able to look at the scripts directory & know exactly what every file is without having to dig into the file itself. Also, it's not quite correct because this wrapper gets called in CircleCI as well as locally.

Reply:
Agreed. test_harness? ci_harness? I'm open to ideas.

Reply:
ci_harness works for me. I think that's a bit more descriptive.